package.json

```diff
@@ -53,6 +53,11 @@
                 "when": "editorTextFocus"
             },
             {
+                "key": "Delete",
+                "command": "extension.vim_delete",
+                "when": "editorTextFocus"
```

Comment:
We are reimplementing delete on in Normal mode, so you may want to check Vim state here like https://github.com/VSCodeVim/Vim/blob/master/package.json#L68